protected/views/project/sync-workspaces-form.php

```diff
@@ -0,0 +1,79 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * @var \prime\components\View $this
```

Comment:
I personally prefer the use statements above the docbloc so the docbloc does not need FQNs